app/assets/stylesheets/content/_select2.scss

```diff
@@ -200,3 +186,7 @@ $se2-width: 100%;
     }
   }
 }
+
+.select2-input {
+  height: rem-calc($se2-input-height) !important;
```

Comment:
@floriank 

> Is this a new rule/configuration not read or a result of your recent discussion

I think @Azure7111 was referring to our discussion about `!important` in a previous PR.